support/scheduler-client/client.go

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright 2017 Dell Inc.
```

Comment:
You are not from Dell, if you are not extending other people work, put your copyright info.